### DISCUSSION TITLE
what is standard init?

### DISCUSSION THREAD

**Discussion by leotohill** [JanDeDobbeleer/oh-my-posh#2220] [Troubleshoot] [answered]
(edited)
I've installed v 7.77 on powershell (not windows powershell) in Terminal on Windows 11.

The only doc on init is under https://ohmyposh.dev/docs/installation/customize which starts by saying "The standard initialization sets Oh My Posh' [default theme])"

What is "the standard initialization?"  This doesn't appear to be documented.  Is it simply the custom initialization, as described on this page, but without the --config parameter? 

IOW there must be a post-installation step to configure, when customization is not needed.  How do I activate omp with a default theme?

BTW I just realized that "config" and "theme" are pretty much synonymous. Right?  There's nothing in a config that isn't part of the theme, and there's nothing in a theme that isn't part of the config.  It might be good to drop/deprecate the "--config" parameter in favor of "--theme".  That would have helped me.

**Comment by JanDeDobbeleer** [comment, marked as answer]
A theme is a predefined config. That's all. I'll see if that can be clarified. Themes come from the days where there wasn't a config.

The default theme is what you get when following the [prompt](https://ohmyposh.dev/docs/installation/prompt) page.

**Reply by leotohill** [reply]
Thanks for the clarification.  

Now I feel dumb for not seeing the prompt page.  Somehow I skipped over that.  It would stand out if, in the doc, if that nav entry was titled "set your prompt" or even "activate your prompt".  

Also note that the app itself gives an invalid url for that doc.  It gives:
"See the documentation to initialize your shell: https://ohmyposh.dev/docs/prompt."
should be "...docs/installation/prompt."

Regarding config vs theme, I'd argue for dropping any distinction.  In the case where any distinction is needed, say "predefined theme" vs "custom theme".   

Thanks for all.

**Reply by lewis-yeung** [reply]
The URL in the CLI help text has been corrected in the recent release v7.78.0. ✅
